rclpy/rclpy/utilities.py

```diff
@@ -15,6 +15,14 @@
 import os
 import sys
 import threading
+from typing import Optional
+
+if sys.version_info < (3, 9, 0):
+    from typing import List
+    from typing import Sequence
+else:
+    List = list
+    from collections.abc import Sequence
```

Comment:
@sloretz can you enlighten me why this is added for the future reference?

Reply:
Ah, I saw `typing.List` and `typing.Sequence` were deprecated in Python 3.9, and assumed it would result in a deprecation warning.

Some local testing and a glance at the [cpython source code](https://github.com/python/cpython/blob/8fe1a88e6ac3df20ad6bb96dd7e25aa18f2f438a/Lib/typing.py#L2705) show's there's no deprecation warning, so I'll get rid of this if/else logic.

Reply:
removed in 0ad3a49

Reply:
@sloretz thanks for the information!